notification/twilio/voice.go

```diff
@@ -638,3 +597,56 @@ func (v *Voice) FriendlyValue(ctx context.Context, value string) (string, error)
 	}
 	return libphonenumber.Format(num, libphonenumber.INTERNATIONAL), nil
 }
+
+// buildMessage is a function that will build the VoiceOptions object with the proper message contents
+func buildMessage(prefix string, msg notification.Message) (opts *VoiceOptions, err error) {
+	if prefix == "" {
+		return nil, errors.New("No prefix provided")
```

Comment:
Small change to lowercase the "no"

Also not sure where this falls in the call chain above as to what the error would be logged/outputted as but should it have a bit more context like `build message error: no prefix provided`?

Reply:
done.  changed to `buildMessage error: no prefix provided`